bundles/net.tourbook/src/net/tourbook/ui/views/rawData/RawDataView.java

```diff
@@ -4953,6 +4965,22 @@ public void run(final IProgressMonitor monitor) throws InvocationTargetException
 
                monitor.worked(1);
             }
+
+            if (invalidFiles != null) {
+
+               for (final String invalidFilePath : invalidFiles) {
+
+                  deleteFile(
+                        deletedFiles,
+                        notDeletedFiles,
+                        Paths.get(invalidFilePath).getParent().toString(),
+                        Paths.get(invalidFilePath).getFileName().toString(),
+                        TourLogState.EASY_IMPORT_DELETE_DEVICE);
+
+                  monitor.worked(1);
```

Comment:
monitor.worked(1) moves the progressbar 1 part forward, therefore it is necessary that the progressbar max value is set correctly. 

But I also discovered that in this line https://github.com/wolfgang-ch/mytourbook/blob/FJBDev/bundles/net.tourbook/src/net/tourbook/ui/views/rawData/RawDataView.java#L4932 the monitor.worked(1) is missing.

Because deleting files are very fast, the wrong progressbar max value will not be noticed.